src/geefetch/cli/download_implementation.py

```diff
@@ -453,29 +452,31 @@ def download_custom(config_path: Path, custom_name: str) -> None:
 
 def download_all(config_path: Path) -> None:
     """Download all configured satellites."""
-    config = load(config_path)
-    if config.s1 is not None:
+    config = load(config_path, add_missing_sats=False)
+
+    # Check which satellites were actually configured by the user
```

Comment:
So this is to check for configuration that contain e.g. `s1: null` for the sole purpose that `download_all` downloads s1  ? Nice I missed this was not working correctly before